x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -45,15 +45,24 @@ export function getTemplate({
   pipelineName,
   packageName,
   composedOfTemplates,
+  hidden,
 }: {
   type: string;
   templateName: string;
   mappings: IndexTemplateMappings;
   pipelineName?: string | undefined;
   packageName: string;
   composedOfTemplates: string[];
+  hidden?: boolean | false;
```

Comment:
Doesn't `boolean | false` make the field non-optional for TS?

Reply:
the field can still be optionally passed - take a look at this unit test case:
https://github.com/elastic/kibana/pull/85703/files#diff-f1bfa3ccb84f6e245a1f2a762eceeb3f2372bd454578568ce84a43c9c2c47fffR85

I'm thinking that we can drop the `| false` part or update it to `| undefined`

Reply:
I'd prefer dropping the `| false` then.

Reply:
updated in both places